### ISSUE TITLE
training convergence and loss value

### ISSUE THREAD

**Issue by Junhojuno** [lee-man/movenet#12]
Thanks for the good implementation material. 
I am currently implementing MoveNet Lightning with Tensorflow and training with Custom Dataset.
(single pose setting)
However, it is not trained enough, and it is not overfitting even with a small amount of data.

The training dataset I have configured has only one object in the image. 
However, the proportion of objects in the image is relatively small (there is a lot of background)
(specifically, 1920x1080 images & most of objects are located in center, there is a lot of blank space on the left and right of the object)

First, I wonder if there were any problems with poor convergence while learning.
Also, I am curious to what extent the loss value converges in the learning process.
